### DISCUSSION TITLE
C++20 modules example

### DISCUSSION THREAD

**Discussion by rdong8** [doxygen/doxygen#11088] [Q&A] [answered]
Is there an example for documenting C++20 modules? I'm having some trouble getting this working.

**Comment by albert-github** [comment]
I don't directly know of an example, but I think it would be better that you show what you tried so far and where your troubles are (might not be related to modules at all).
So it will be possible / easier top give some help.
In general:
- Can you please attach a, small, self contained example (source+configuration file in a, compressed, tar or zip file!) that allows us to reproduce the problem? Please don't add external links as they might not be persistent (also references to GitHub repositories are considered non persistent).
- Please also specify the **full** doxygen version used (`doxygen -v`).

**Comment by albert-github** [comment, marked as answer]
In your project (assuming it is https://github.com/rdong8/cpp_project) you have
```
FILE_PATTERNS = *.h
```
but the documented code is in ccm files, so shouldn't it be:
```
FILE_PATTERNS =  *.ccm
```
there are no `*.h` files in your project at all.

**Reply by albert-github** [reply]
Based on https://github.com/rdong8/cpp_project/issues/2#issuecomment-2306361769  does this mean that this discussion can be closed as well?
